app/processor.rb

```diff
@@ -4,15 +4,20 @@
 class Processor
   def initialize
     @logger = CacheClearingService.config.logger
+    @varnish_clearer = VarnishClearer.new(logger)
+    @fastly_clearer = FastlyClearer.new(logger)
   end
 
   def process(message)
-    VarnishClearer.new(logger).clear_for(message.payload.fetch("base_path"))
+    base_path = message.payload.fetch("base_path")
+
+    varnish_clearer.clear_for(base_path)
+    fastly_clearer.clear_for(base_path)
```

Comment:
Should you be clearing all the items in the routes array rather than base_path?

Reply:
Good point, I think we'll add that in a separate PR.